dash-pipeline/tests/requirements.txt

```diff
@@ -1,2 +1,3 @@
 snappi==0.7.38
 pytest==6.0.1
+ptf==0.9.3
```

Comment:
Please revert, see main comments for reason.

Reply:
removed